tasks/build.js

```diff
@@ -27,18 +27,18 @@ bozon.task('scripts:renderer', function () {
 });
 
 bozon.task('config', function () {
-  bozon.src('package.json').pipe(jsonEditor({
+  return bozon.src('package.json').pipe(jsonEditor({
     settings: settings.get()
   })).pipe(bozon.dest());
 });
 
 bozon.task('images', function () {
-  bozon.src('images/**/*').pipe(bozon.dest('images'));
+  return bozon.src('images/**/*').pipe(bozon.dest('images'));
 });
 
-bozon.task('compile', function () {
+bozon.task('compile', function (callback) {
   settings = new bozon.Settings();
-  runSequence('scripts:main', 'scripts:renderer', 'styles', 'html', 'images', 'config');
+  runSequence.apply(this, ['scripts:main', 'scripts:renderer', 'styles', 'html', 'images'].concat(bozon.hooks, 'config', callback));
```

Comment:
I really like the idea of adding hooks to bozon gulp tasks, but not sure about placing them into this specific place in a sequence. Wouldn't it be more flexible to allow users to define where their tasks should take place? Maybe something like: 
```javascript
task = bozon.task('whatever', function () { ... });
bozon.insertAfter('images', task);
```

or even

```javascript
bozon.runAfter('images', function () { ... });
bozon.runBefore('config', function () { ... });
```